### PR TITLE
Kb id button and line limitation

### PR DESCRIPTION
**Pull request by itIsJim**

### CODE REVIEW

File: content/en/seachat/seachat-manual/03-add-knowledge/09-add-webpage-link-in-answers.md [hunk 1]
```diff
@@ -92,18 +92,28 @@ Once there is information in the knowledge base, the agent will start using this
...
+    <a href="/images/seachat/en/knowledge-advanced-features/url-button/kb-id-problem.png" target="_blank">
+    <img width="100%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/seachat/en/knowledge-advanced-features/url-button/kb-id-problem.png" alt="">
     </a>
 </div>
+    <p style="margin-top: 20px; font-size: 15px">Message Cutoff due to Message Limitation</p>
 </div>
-
 <br/>
 
+Here is a summary of the current limits for our button templates and postback buttons:
```
Comment (xuchen) [inline content/en/seachat/seachat-manual/03-add-knowledge/09-add-webpage-link-in-answers.md:107]: this is still not clear to me: is the limit from webchat or Line?

Originally when we introduced this, it's supposed to resolve the Line limitation. But here it doens't mention Line at al!!!

**Review by xuchen — CHANGES_REQUESTED** [review]
See the Line comment.

Then also in the "Line" section: seachat/seachat-manual/04-channels/05-install-to-line/#-faq

please insert something like: "The Line button cuts the messages short. How do I post the whole message?" Then explain Line's limit on button message length.

File: content/en/seachat/seachat-manual/03-add-knowledge/09-add-webpage-link-in-answers.md (hunk 1 above)
Comment (itIsJim) [reply content/en/seachat/seachat-manual/03-add-knowledge/09-add-webpage-link-in-answers.md:107]: I will move this limitation to the section of Line and reference it from here.

File: content/en/seachat/seachat-manual/04-channels/05-install-to-line.md [hunk 2]
```diff
@@ -117,6 +117,84 @@ You can use SeaChat as an AI assistant providing useful information, capable of
...
+    * Messaging API's Reply API.
+
+<br/>
+
+
+## Limits of LINE Button Messages
+
+When users are using the LINE channel with SeaChat, they may encounter issues where the button message is cut off when the message button is clicked. This is due to the character limit for the LINE button message.
+
+SeaChat has a solution for this issue. Utilize the feature of KB ID to avoid the message being cut off. Please check out our wiki about [KB ID](https://wiki.seasalt.ai/seachat/seachat-manual/03-add-knowledge/09-add-webpage-link-in-answers/#kb-ids) for more information on how to avoid the message being cut off.
```
Comment (xuchen) [inline content/en/seachat/seachat-manual/04-channels/05-install-to-line.md:182]: could you please move this line to the end of this section, after Line 196?

**Review by xuchen — APPROVED** [review]

**Pull request merged**